static/js/modules/helpers.js

```diff
@@ -82,6 +82,31 @@ Handlebars.registerHelper('panelRow', function(label, options) {
   );
 });
 
+Handlebars.registerHelper('entityUrl', function(entity, options) {
+  if (options.hash.query) {
+    var query = {
+      cycle: options.hash.query.cycle || null,
+      election_full: options.hash.query.election_full || null
+    };
+  }
+  var id = entity.candidate_id || entity.committee_id;
+  var url = buildAppUrl([options.hash.type, id], query);
```

Comment:
Hound is getting confused here, but for good reason.  There's a mechanism known as variable hoisting in JavaScript; what this means is any variable defined in a method anywhere is going to be instantiated first at the beginning of the function at run-time, regardless of where it's used.  This, combined with JavaScript's block and scoping rules are what are triggering the alarms here and had me tripped up at first too.

What you can do is define the variables in the method first and set them after.  This helps account for the variable hoisting and ensures the proper scope is recognized.  At the beginning of the method, I would do this before anything else:

`var id, query, url;`

And then remove the `var` statements elsewhere in the method.  More information on variable hoisting can be found [here](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) and [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting).